src/Adapters/ADTDataHistoryAdapter.ts

```diff
@@ -57,14 +57,15 @@ export default class ADTDataHistoryAdapter implements IADTDataHistoryAdapter {
         );
         /**
          * Check if class has been initialized with CORS enabled or if origin matches dev or prod explorer urls,
-         * override if proxy is forced by feature flag
+         * override if CORS is forced by feature flag
          *  */
         this.useProxy =
-            useProxy ||
-            !validateExplorerOrigin(window.origin) ||
-            localStorage.getItem(
-                LOCAL_STORAGE_KEYS.FeatureFlags.Proxy.forceProxy
-            ) === 'true';
+            (useProxy || !validateExplorerOrigin(window.origin)) &&
+            !(
+                localStorage.getItem(
+                    LOCAL_STORAGE_KEYS.FeatureFlags.Proxy.forceCORS
+                ) === 'true'
+            );
```

Comment:
This would read better as `localStorage.getItem( LOCAL_STORAGE_KEYS.FeatureFlags.Proxy.forceCORS ) !== 'true'` removing the wrapping parenthesis and negation.